test/langtools/jdk/javadoc/doclet/testBreakIterator/TestBreakIterator.java

```diff
@@ -76,5 +76,10 @@ public void test() {
                 """
                     <div class="block">A constant indicating that the keyLocation is indeterminate
                      or not relevant.</div>""");
+
+        checkOutput("pkg/BreakIteratorTest.html", true,
+                """
+                    <div class="block">Inline tags <i><a href="../index-all.html">extending
+                     beyond the first sentence.</a></i></div>""");
```

Comment:
I have a question:
Why doesn't it include ` Tags are closed here`? Is it intentional or a bug?

Reply:
This piece of HTML is from the method summary table which only displays the first sentence of the doc comment (which is the root of the problem fixed by this PR).

Reply:
Got it. Thanks for your answer.